tests/exporters/test_onnx_export.py

```diff
@@ -375,7 +387,6 @@ def test_pytorch_export_for_stable_diffusion_models(self, model_name):
             validate_models_outputs(
                 models_and_onnx_configs=models_and_onnx_configs,
                 onnx_named_outputs=onnx_outputs,
-                atol=1e-3,
```

Comment:
why remove?

Reply:
It allows to take the atol of each of the indivual component (specified in each onnx config)

Reply:
This was removed as the VAE encoder has a much bigger atol, I will add it back now that the decoder test is adapted !